x-pack/plugins/alerts/server/usage/alerts_usage_collector.ts

```diff
@@ -57,7 +57,7 @@ export function createAlertsUsageCollector(
       try {
         const doc = await getLatestTaskState(await taskManager);
         // get the accumulated state from the recurring task
-        const state: AlertsUsage = get(doc, 'state') as AlertsUsage;
+        const { runs, ...state } = get(doc, 'state') as AlertsUsage & { runs: number };
```

Comment:
@elastic/kibana-alerting-services, same comment as earlier: if we want to include it, we'll need to update `AlertsUsage` and the schema.